packages/renderer/src/lib/preferences/docker-compat/PreferencesDockerCompatibilitySocketMappingStatus.svelte

```diff
@@ -16,7 +16,8 @@ let isLinux = $state(false);
 
 let dockerSocketMappingStatusInfo: DockerSocketMappingStatusInfo | undefined = $state(undefined);
 
-let engineType: 'kubernetes' | 'podman' | 'docker' | undefined = $state(undefined);
+let engineType: 'kubernetes' | 'podman' | 'docker' | 'lima' | undefined = $state(undefined);
```

Comment:
question (non-blocking): is the type here `DockerSocketServerInfoType` ? or something different, because I see `undefined`, and not `unknown` ?

If the type derive from DockerSocketServerInfoType like `DockerSocketServerInfoType | undefined` would be nice to use it

Reply:
The name "engineType" is not ideal, since it actually refers to the serverType.

i.e. the string is just converted from "Podman Machine OS" to "Podman Engine"

Reply:
Humm okay thanks for the details

Reply:
Note that "kubernetes" is unused